### PR TITLE
fix: [PIE-4788] : Environment Details UX Review

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
### Summary

Updated followings for the EnvironmentV2 as per UX Review:
-  Icon size to be made 16px for plus icons in buttons
-  Configuration and other tabs text made bold when highlighted
- Name - text to be made bold
- Environment Type - text to match name id description tag
- Increase padding on the left in the environment details page to xxlarge

#### Screenshots
Screenshot 2022-08-12 at 6 27 57 PM
Screenshot 2022-08-12 at 6 28 32 PM
Screenshot 2022-08-12 at 6 28 44 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10562)

File: src/modules/75-cd/components/EnvironmentsV2/PageTemplate/PageTemplate.tsx [hunk 1]
```diff
@@ -234,7 +234,7 @@ export default function PageTemplate({
           <RbacButton
             intent="primary"
             icon="plus"
-            iconProps={{ size: 12 }}
+            iconProps={{ size: 16 }}
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/EnvironmentsV2/PageTemplate/PageTemplate.tsx:237]: What if we removed this props entirely? It should default to 16 I believe

Comment (pankajsingh-harness) [reply src/modules/75-cd/components/EnvironmentsV2/PageTemplate/PageTemplate.tsx]: Done

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
trigger cypress

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by udayvunnam — APPROVED** [review]

**Pull request merged**